Cue/app/tabs/library/LibraryHome.js

```diff
@@ -80,21 +80,20 @@ class LibraryHome extends React.Component {
   }
 
   _refresh = () => {
+    this.setState({refreshing: true})
     this.props.onSyncLibrary(this.props.localChanges).then(failedSyncs =>{
       if (failedSyncs && failedSyncs.length) {
         this.props.navigator.push({failedSyncs})
+        this.setState({refreshing: false})
```

Comment:
I don't think it really matters, but I'd call `setState` before pushing the navigator route.